flocker/sample/test_sample.py

```diff
@@ -0,0 +1,33 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+Sample tests for skipping.
+"""
+from flocker.testtools.skips import skipUnless, skipIf, SkipTest
+from twisted.trial.unittest import TestCase
+
+skip_this_test_with_skipUnless = skipUnless(
```

Comment:
https://github.com/ClusterHQ/flocker/pull/1050/files#r23014698 gave an example of the environment variable being used.

It is not that the skip is ignored, however. I thought that debugging would be much easier if the test failed when it was otherwise meant to skip.